internal/pkg/service/buffer/store/record_test.go

```diff
@@ -25,14 +25,14 @@ func TestStore_CreateRecord(t *testing.T) {
 	assert.NoError(t, err)
 
 	csv := []string{"one", "two", `th"ree`}
-	record := key.NewRecordKey(projectID, receiverID, exportID, "file1", "slice1", now)
+	record := key.NewRecordKey(projectID, receiverID, exportID, now, now, now)
 	err = store.CreateRecord(ctx, record, csv)
 	assert.NoError(t, err)
 
 	// Check keys
 	etcdhelper.AssertKVs(t, store.client, `
 <<<<<
-record/1000/receiver1/export1/file1/slice1/2006-01-02T08:04:05.000Z_%c%c%c%c%c
+record/1000/receiver1/export1/2006-01-02T08:04:05.000Z/2006-01-02T08:04:05.000Z/2006-01-02T08:04:05.000Z_%c%c%c%c%c
```

Comment:
Na vylucenie nejakej banalnej chyby, ze sa niekde omylom nepomiesali premenne: by som nezvolil `now, now, now`, ale `time1, time2, time3`.

Reply:
Je mi jasne, ze prva slice bude mat `fileID` a `sliceID` rovnake, ale dalsie nie. A ked to otestujeme pre rozne hodnoty, tak je jasne, ze to funguje aj pre rovnake.

Reply:
Inak, nerozmyslal si, ze by si z `RecordKey` vyhodil `FileID`?  (Iba tu v record, nie v SliceKey -> string).

Ono je to zrejme az teraz v tomto teste.
SliceID je vzdy unikatne, kedze nikdy nevytvarame viacero slices v jednom exporte naraz.

A ku recordu sa vzdy pristupuje na urovni slice.
Tj. idem nahravat slice, tak si najdem vsetky records k danej slice.

-----

Tak len taky navrh ako skratit tento kluc, ktory mi napadol az teraz.